includes/class-convertkit-output.php

```diff
@@ -327,20 +327,32 @@ public function append_form_to_content( $content ) {
 		}
 
 		// If here, we have a ConvertKit Form.
-		// Append form to Post's Content.
-		$content = $content .= $form;
+		// Append form to Post's Content, based on the position setting.
+		$form_position = $this->settings->get_default_form_position( get_post_type( $post_id ) );
+		switch ( $form_position ) {
```

Comment:
Keeping this as a `switch` statement as future PR's will add support for other positions (such as after a given number of paragraphs, a specific element such as an image or video etc).